charts/tf-controller/templates/crds.yaml

```diff
@@ -5269,6 +5269,17 @@ spec:
                 description: RefreshBeforeApply forces refreshing of the state before
                   the apply step.
                 type: boolean
+              remediation:
+                description: Remediation holds the remediation configuration for when
+                  the reconciliation fails. The default is to not perform any action.
+                properties:
+                  retries:
+                    description: Retries is the number of retries that should be attempted
+                      on failures before bailing. Defaults to '0', a negative integer
+                      equals to unlimited retries.
+                    format: int64
+                    type: integer
+                type: object
               retryInterval:
```

Comment:
Yes I said the same, but otherwise we would have to duplicate that field, check if either of them is set, use that value, set priorities if both set, etc.